_overviews/scala3-book/control-structures.md

```diff
@@ -60,28 +106,57 @@ else if x == 0 then
 else
   println("positive")
 ```
+{% endtab %}
+{% endtabs %}
 
-You can optionally include an `end if` statement at the end of each expression, if you prefer:
+### `end if` statement
+
+<blockquote class="help-info">
+<i class="fa fa-info"></i>&nbsp;&nbsp;This is only new in Scala 3.
```

Comment:
I would say maybe “This is only supported in Scala 3”, or “This is new in Scala 3, and not supported in Scala 2”, but I am not sure about “This is only new in Scala 3”.